README.md

```diff
@@ -1,25 +1,19 @@
 # RPM for drush
 
-This repository builds an RPM for drush.
-
-## Relevant drush RPM names
-
-- drush: Legacy RPM containing drush and site-audit
-- drush-0.x: RPM containing only drush (version 0.x)
-- site-audit-0.x: RPM containing only site-audit 0.x
+This repository builds RPMs for drush.
 
 The RPM filename built by this repository is:
 ```
-drush-0.x-release-0.22.0-01458238016.git4602714.x86_64.rpm
+drush-x-release-8.1.2-1466798938.2339402.x86_64.rpm
```

Comment:
I missed something, and said the wrong thing.  We want to filename of the rpm to be `drush-8.x-release...`.  It should still install to the folder `/opt/pantheon/drush-8`, though.  (Only add the -x to the rpm filename; leave everything else just `-8`).